pkg/0e89154b-a448-4fea-a115-169fb56c0cb9/package.lp

```diff
@@ -0,0 +1,185 @@
+(librepcb_package 0e89154b-a448-4fea-a115-169fb56c0cb9
+ (name "SOT353")
+ (description "aka SC70-5, SC88A, TSSOP-5, UMT5\n\nhttps://en.wikipedia.org/wiki/Small-outline_transistor")
+ (keywords "sot353,sc70-5,sc88a,tssop-5,umt5")
+ (author "Danilo B.")
+ (version "0.1")
+ (created 2018-11-01T19:29:35Z)
+ (deprecated false)
+ (category b77c2754-fc91-4e9e-949e-5568a4bee0fe)
+ (pad 34f43b73-db56-4649-8904-0d258eeb7bfe (name "1"))
+ (pad a2b521d2-b58b-45b4-944a-65bd6e6e158a (name "2"))
+ (pad fa0eed07-e0ca-4ed6-bb33-4c54a46bb925 (name "3"))
+ (pad dc18d314-08ec-45f8-bd81-8b83857d1402 (name "4"))
+ (pad a837e245-00a8-45f7-befe-8aaafc1001f8 (name "5"))
+ (footprint 071e06d0-7e9d-4667-8a85-ab3dad74a72a
+  (name "default")
+  (description "")
+  (pad 34f43b73-db56-4649-8904-0d258eeb7bfe (side top) (shape rect)
+   (position -0.9 0.75) (rotation 0.0) (size 0.6 0.6) (drill 0.0)
+  )
+  (pad a2b521d2-b58b-45b4-944a-65bd6e6e158a (side top) (shape rect)
+   (position -0.9 0.0) (rotation 0.0) (size 0.6 0.4) (drill 0.0)
+  )
+  (pad fa0eed07-e0ca-4ed6-bb33-4c54a46bb925 (side top) (shape rect)
+   (position -0.9 -0.75) (rotation 0.0) (size 0.6 0.6) (drill 0.0)
+  )
+  (pad dc18d314-08ec-45f8-bd81-8b83857d1402 (side top) (shape rect)
+   (position 0.9 -0.75) (rotation 0.0) (size 0.6 0.6) (drill 0.0)
+  )
+  (pad a837e245-00a8-45f7-befe-8aaafc1001f8 (side top) (shape rect)
+   (position 0.9 0.75) (rotation 0.0) (size 0.6 0.6) (drill 0.0)
+  )
+  (polygon f1ab1098-3aeb-4172-b9e3-25f7b70370a8 (layer top_documentation)
+   (width 0.1524) (fill false) (grab_area true)
+   (vertex (position 0.63 1.0) (angle 0.0))
+   (vertex (position -0.63 1.0) (angle 0.0))
+   (vertex (position -0.63 -1.0) (angle 0.0))
+   (vertex (position 0.63 -1.0) (angle 0.0))
+   (vertex (position 0.63 1.0) (angle 0.0))
+  )
+  (polygon f1ab1098-3aeb-4172-b9e3-25f7b70370a8 (layer top_placement)
```

Comment:
This polygon makes the silkscreen overlapping with pads, which is bad. You should change it to what IPC7351 suggests (the inner rectangle):
![grafik](https://user-images.githubusercontent.com/5374821/47881213-cbd9b980-de25-11e8-93c0-76d0b120263b.png)

Same for the other footprint.

Reply:
Fixed.

Reply:
It's still extremely near to the pads and thus will probably overlap with the solder mask. So it needs to be even smaller. I would suggest at least 0.15mm clearance between pad and soldermask.

And now you also decreased the size of the polygon on the documentation layer :( Documentation should show the *real* size of the package, no matter if it overlaps with pads because it's not printed on silkscreen.

Reply:
> Documentation should show the real size of the package, no matter if it overlaps with pads because it's not printed on silkscreen.

Can we have these things documented in guidelines? You seem to have a quite clear picture of how these symbols and footprints should look like, so please write them down. Otherwise I'll do all the work three times :slightly_smiling_face:

Reply:
> Can we have these things documented in guidelines?

Sure, I just didn't find the time yet to do so...

In the mean time you can just take a look at the already existing library elements and do it the same way. For some of your new elements there are very similar elements already existing:
- SOT353 -> See SOT23-3
- ATtiny85 -> See STM32F103C
- Naming conventions (e.g. Title Case for element names), see *any* existing element

In addition, for packages please always take a look at IPC7351. I also don't know what is written there exactly, so I also take a look at it every time I review a package ;)

Reply:
Done. I also changed the inner rectangle style to "filled", since that seems to be the IPC recommendation for SOT343.